x-pack/plugins/security_solution/common/endpoint/data_generators/base_data_generator.ts

```diff
@@ -102,7 +102,7 @@ export class BaseDataGenerator<GeneratedDoc extends {} = {}> {
   }
 
   protected randomVersion(): string {
-    return [6, ...this.randomNGenerator(10, 2)].map((x) => x.toString()).join('.');
+    return [7, ...this.randomNGenerator(20, 2)].map((x) => x.toString()).join('.');
```

Comment:
Make the version generator possible to be 7.14.0+ to ensure fake docs will pass the version check.